docs/getting_started/requirements.md

```diff
@@ -33,7 +33,7 @@ event MPM is recommended, if you need to use prefork you also need the `mod_php`
 
 - MariaDB 10.3+
 - MySQL 8.0
-- PostgreSQL 10+
+- PostgreSQL 14
```

Comment:
I'm not too happy about this 😑 For one, we might want to wait for the resolution of https://github.com/ezsystems/ezpublish-kernel/pull/3152 even if it isn't the same thing.

So we say "we recommend this version or higher within the same major release". That's problematic here since postgres mainly uses major versions. If we truly do recommend 14 or higher then sure, fine. But do we? I thought we just recommend currently supported postgres versions, which would be `11 - 15`. I think I would prefer that, once it is QA-ed 😉

Reply:
I'm staring to think we should drop the "+" sign all together and replace it with lists explicitly stating (major) versions.

Reply:
⬆️ 🔥  You have a point.